TestProject/Controllers/Web/WebViewController.m

```diff
@@ -0,0 +1,82 @@
+//
+//  WebViewController.m
+//  TestProject
+//
+//  Created by Developer on 7/20/17.
+//  Copyright © 2017 Developer. All rights reserved.
+//
+
+#import "WebViewController.h"
+
+@interface WebViewController ()
+
+@property (weak, nonatomic) IBOutlet UIWebView *webView;
+@property (weak, nonatomic) IBOutlet UIActivityIndicatorView *activityIndicator;
+@property (weak, nonatomic) IBOutlet UIBarButtonItem *backButtonItem;
+@property (weak, nonatomic) IBOutlet UIBarButtonItem *forwardButtonItem;
+@property (weak, nonatomic) IBOutlet UIBarButtonItem *refreshButtonItem;
+@property (weak, nonatomic) IBOutlet UIBarButtonItem *actionButtonItem;
+
+@end
+
+@implementation WebViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    NSURL *url = [NSURL URLWithString:@"http://apple.com"];
+    NSURLRequest *request = [NSURLRequest requestWithURL:url];
+    [self.webView loadRequest:request];
+    
+}
+
+#pragma mark - Actions
+
+- (IBAction)actionBackButton:(UIBarButtonItem *)sender {
+    if ([self.webView canGoBack]) {
+        [self.webView goBack];
+    }
+}
+
+- (IBAction)actionForwardButton:(UIBarButtonItem *)sender {
+    if ([self.webView canGoForward]) {
+        [self.webView stopLoading];
+        [self.webView goForward];
+    }
+}
+
+- (IBAction)actionRefreshButton:(UIBarButtonItem *)sender {
+    [self.webView reload];
+}
+
+- (IBAction)actionActionButton:(UIBarButtonItem *)sender {
+    
+}
+
+#pragma mark - UIWebViewDelegate
+
+- (BOOL)webView:(UIWebView *)webView shouldStartLoadWithRequest:(NSURLRequest *)request navigationType:(UIWebViewNavigationType)navigationType {
+
+    return YES;
+}
+
+- (void)webViewDidStartLoad:(UIWebView *)webView {
+    [self.activityIndicator startAnimating];
+    
+    [self.refreshButtonItem setTitle:@"Stop"];
+}
+
+- (void)webViewDidFinishLoad:(UIWebView *)webView {
+    [self.activityIndicator stopAnimating];
+    
+    [self.refreshButtonItem setTitle:@"refresh"];
+
+    self.backButtonItem.enabled = [self.webView canGoBack];
+    self.forwardButtonItem.enabled = [self.webView canGoForward];
+}
+
+- (void)webView:(UIWebView *)webView didFailLoadWithError:(NSError *)error {
+    
```

Comment:
The logic of this method should be the same as previous one.
But do not duplicate code.